### ISSUE TITLE
add existing question to archive catalog data-usage/costs-dataset/storage_non_personnel

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#51]
**Describe the request**  
The storage colleagues lack precise information on whether funding has been applied for for data storage.

**Propose what exactly should be changed**  
~~Either a new question should be generated or~~ The existing question `data-usage/costs-dataset/storage_non_personnel`~~from the general UA-Ruhr catalog~~ should be ~~changed and then~~ added to the archiving catalog.

**Original setting or content**   
~~Concrete questions about subsidies for data management are not yet asked:~~

~~*question text ger*~~ 
~~"In welcher Förderlinie und/oder welchem Förderprogramm wird das Projekt gefördert?"~~

**Proposed setting or content**   
~~New question: "Wurden Fördermittel für die Datenhaltung beantragt?"~~

*question text ger*  
~~or alternatively change question in "In welcher Förderlinie und/oder welchem Förderprogramm wird das Projekt gefördert und wurden dabei Fördermittel für die Datenhaltung beantragt?"~~

**Additional context**

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ] add question text (en and ger) in archive catalogue

**Comment by smohex** [comment]
TUDO: Add exactly there
UDE: change question as suggested

**Comment by jmfrenzel** [comment]
data-usage/costs-dataset/storage_non_personnel

~~data-usage/costs-dataset/usage_non_personnel~~

**Comment by jmfrenzel** [comment]
Question to be added via #75

**Comment by ClaraRosalie** [comment]
this is ua ruhr specific content and therefore not implemented in upstream
